### ISSUE TITLE
Able to block as well as follow somebody at the same time

### ISSUE THREAD

**Issue by thehydrogen** [benawad/dogehouse#2678]
You are able to follow someone as well as have them blocked.

**To Reproduce**
1. Go to a user profile
2. Click on 'follow'
3. Click on 'block'

**Expected behavior**
Blocking would remove your follow.

**Additional context**
If this is intended, my mistake.

**Comment by DanLew** [comment]
I don't think this is intended. It looks like in the user block method [here](https://github.com/benawad/dogehouse/blob/d81925c5696939bca15e8cc6dff1a9aad56c63d6/kousa/lib/kousa/user_block.ex#L6) it should be removing the follow when blocking the user.
